Task_TheFlexGame/index.html

```diff
@@ -1,15 +1,41 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Flex Game</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-
-    <!-- Start coding from the lines below for FLEX GAME TASK; DO NOT TOUCH the code above -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Flex Game</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="index.css" />
+</head>
+
+<body style="margin: 0; padding: 0">
+    <div>
+        <a href="../index.html">Go Back</a>
+    </div>
+
+    <!-- Start coding from the lines below for FLEX GAME TASK; DO NOT TOUCH the code above -->
+    <div class="container">
+        <div class="menu">
+            <div class="menu-title">Menu</div>
+            <div class="menu-boxes">1</div>

```

Comment:
Consider adding more space between each content box for better visual separation and organization.